test/verify/machineslib.py

```diff
@@ -1134,7 +1134,11 @@ def installWithErrorTest(dialog):
         # This is applicable only for the following test so let's keep it last,
         # in order to allow the rest of the tests to run faster with QEMU KVM
         # Run modprobe -r in retry loop because it fails sometimes with 'Module kvm is in use'
-        self.machine.execute("for i in 1 2 3 4 5; do modprobe -r kvm_intel && modprobe -r kvm && break || sleep 1; done")
+        try:
+            wait(lambda: "Module kvm is in use" not in self.machine.execute("modprobe -r kvm_intel && modprobe -r kvm_intel && modprobe -r kvm"))
+        except Error:
+            print(self.machine.execute("pgrep -af qemu || true"))
```

Comment:
I. e. let's let this do `lsof /dev/kvm` in addition.

Reply:
As we discussed over IRC pmcdkvm does not open /dev/kvm so fuser command is not showing anything helpfull